src/protocol/server.c

```diff
@@ -131,15 +131,15 @@ protocolServerProcess(
             {
                 // Read command
                 KeyValue *commandKv = jsonToKv(ioReadLine(protocolServerIoRead(this)));
-                const String *command = varStr(kvGet(commandKv, VARSTR(PROTOCOL_KEY_COMMAND_STR)));
+                const StringId command = strIdFromStr(stringIdBit5, varStr(kvGet(commandKv, VARSTR(PROTOCOL_KEY_COMMAND_STR))));
```

Comment:
Do we specify anywhere that all PROTOCOL_COMMAND_* must be STRID5? What happens if it is stringIdBit6 format?

Reply:
There's no requirement that they be STRID5, it is just slightly more efficient that way since they are smaller.

Reply:
Oops, yeah, for the time being they do need to be STRID5 -- but that's a temporary condition until the binary protocol is in place.

Reply:
6-bit will work when we move the binary protocol but I see no reason why we should. I added comments to specify that these StringIds should be 5-bit in d8902130e0.